common/middleware/set-services.js

```diff
@@ -0,0 +1,26 @@
+const services = require('../services')
+
+const setServices = (req, res, next) => {
+  const servicesRequestContext = {}
```

Comment:
Is the intention of this to cache the services? I think this placement will result it being cleared on each request.